packages/cashier/src/constants/routes-config.tsx

```diff
@@ -34,7 +34,11 @@ const initRoutesConfig = (): TRouteConfig[] => [
             },
             {
                 path: routes.cashier_withdrawal,
-                component: Withdrawal,
+                component: () => (
+                    <CashierLockedChecker>
+                        <Withdrawal />
+                    </CashierLockedChecker>
```

Comment:
Wrapping the `Withdrawal` page with the generic `CashierLockedChecker` component at the route level.